### ISSUE TITLE
Not working although parameters were supplied.

### ISSUE THREAD

**Issue by DMNight6** [R0rt1z2/realme-ota#13]
```
realme-ota -r 0 RMX2195 RMX2195_11.C.10_1100_202201100634 2
usage: main.py [-h] [-r {0,1,2,3}] [-d DUMP] [-o ONLY] [-s {0,1}] [-v {0,1}] product_model ota_version {1,2,3}
main.py: error: the following arguments are required: product_model, ota_version, rui_version
```

**Comment by DMNight6** [comment]
Now it's artifactV1Result invalid
```
./main.py RMX2195 RMX2195_11.C.10_1100_202201100634 2
[2022-02-23 10:36:06.586795] I: Load payload for RMX2195 (RealmeUI V2)
[2022-02-23 10:36:06.588050] I: Wait for the endpoint to reply
[2022-02-23 10:36:06.821388] F: Received invalid response (artifactV1Result is empty)!
```

**Comment by DMNight6** [comment]
Fix by replacing everything with 0 (#5 )
